x-pack/plugins/apm/public/components/app/mobile/service_overview/index.tsx

```diff
@@ -163,114 +164,108 @@ export function MobileServiceOverview() {
-                          {
-                            defaultMessage: 'App version',
-                          }
-                        )}
-                        metric={SERVICE_VERSION}
-                        start={start}
-                        end={end}
-                        kuery={kueryWithMobileFilters}
-                        filters={embeddableFilters}
-                      />
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </EuiPanel>
+              <EuiFlexItem grow={4}>
+                <MobileLocationStats
+                  start={start}
+                  end={end}
+                  kuery={kueryWithMobileFilters}
+                />
               </EuiFlexItem>
             </EuiFlexGroup>
           </EuiFlexItem>
+
+          <EuiFlexItem>
+            <EuiPanel hasBorder={true} color="subdued">
+              <EuiFlexItem grow={false}>
+                <EuiTitle size="xs">
+                  <h2>
+                    {i18n.translate('xpack.apm.serviceOverview.mostUsedTitle', {
+                      defaultMessage: 'Most used',
+                    })}
+                  </h2>
+                </EuiTitle>
+              </EuiFlexItem>
+              <EuiSpacer size="xs" />
+              <EuiFlexGroup direction={rowDirection} gutterSize="s">
+                {/* Device */}
+                <EuiFlexItem>
+                  <EuiPanel hasBorder={true}>
+                    <MostUsedChart
+                      title={i18n.translate(
+                        'xpack.apm.serviceOverview.mostUsed.device',
+                        {
+                          defaultMessage: 'Devices',
+                        }
+                      )}
+                      metric={DEVICE_MODEL_IDENTIFIER}
+                      start={start}
+                      end={end}
+                      kuery={kueryWithMobileFilters}
+                      filters={embeddableFilters}
+                    />
+                  </EuiPanel>
+                </EuiFlexItem>
+                {/* NCT */}
+                <EuiFlexItem>
+                  <EuiPanel hasBorder={true}>
+                    <MostUsedChart
+                      title={i18n.translate(
+                        'xpack.apm.serviceOverview.mostUsed.nct',
+                        {
+                          defaultMessage: 'Network Connection Type',
+                        }
+                      )}
+                      metric={NETWORK_CONNECTION_TYPE}
+                      start={start}
+                      end={end}
+                      kuery={kueryWithMobileFilters}
+                      filters={embeddableFilters}
+                    />
+                  </EuiPanel>
+                </EuiFlexItem>
+                {/* </EuiFlexGroup> */}
+                <EuiSpacer size="s" />
+                {/* <EuiFlexGroup direction={rowDirection} gutterSize="s"> */}
```

Comment:
was this commented out on purpose?

Reply:
Oh, leftovers from when I was moving this around to fix the UI layout, good catch